src/ansible_navigator/ui_framework/field_button.py

```diff
@@ -29,15 +30,15 @@ def full_prompt(self) -> str:
         """no default to add into the prompt for checkbox"""
         return ""
 
-    def validate(self, response: "FieldButton") -> None:
+    def validate(self, response: List) -> None:
```

Comment:
Usually is a bad idea to use container objects without a type for the contained things. 

It is a list of what? I cannot really guess from the signature and description. Can you make it more specific?

Reply:
good catch, thanks, working on it

Reply:
thanks again, in this case it should be a list of either Unknown or a bool indicating the validaoty of the other fields in the form